### PR TITLE
US[1] Added form

### PR DESCRIPTION
**Pull request by MorganeL-pro**
Create the form to show one question and its answers
(files to check : QuestionManager.php, QuestionController.php, AnswerManager.php, routes.php and twig views (index, answers + components)

### CODE REVIEW

**Review by iStyyx — DISMISSED** [review]
Le fonctionnement est bon, le code a été vérifié sans erreurs.

File: src/Model/QuestionManager.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public function update(array $item): bool
+    {
+        $statement = $this->pdo->prepare("UPDATE " . self::TABLE . " SET `title` = :title WHERE id=:id");
+        $statement->bindValue('id', $item['id'], \PDO::PARAM_INT);
+        $statement->bindValue('title', $item['title'], \PDO::PARAM_STR);
+
+        return $statement->execute();
+    }
+
+    public function getRandomQuestion(): array
```
Comment (RomainClair) [inline src/Model/QuestionManager.php]: To be coherent with the AbstractManager, this method should probably be called selectRandomQuestion

**Review by RomainClair — CHANGES_REQUESTED** [review]
The / (index) and /answers HTML pages are invalid according to W3C validator.
US[1] is : "arriver sur un site beau, simple, facile d’utilisation"...
But according to the comments, it seems that this PR is about 3 US : 
1 - I can access the list of questions
2 - I can list the different possible answer of a question
3 - I can see information for a specific question.

PR should be atomic. i.e. : it should be about 1 US only

**Comment by MorganeL-pro** [comment]
We've changed the branch to show only the files used for [US1] feature "the user can see a page with a question and its answers displayed randomly.

**Review by iStyyx — APPROVED** [review]
It seems more clarly, no errors (Sniffer & W3C) and the feature is working for me.

**Review by RomainClair — COMMENTED** [review]
It's a bit strange to have 2 headers and 2 footer on a page.
I guess the idea is to have the nav on the top of the page or on the bottom depending of the user's device...
But wouldn't it be a better idea to just have a single header, footer and a nav outside header and footer and position the nav using CSS ?

**Review by RomainClair — APPROVED** [review]

**Pull request merged**